.github/workflows/downstream_tests.yml

```diff
@@ -7,6 +7,9 @@ on:
 
 jobs:
   altair:
+    env:
+      UV_SYSTEM_PYTHON: true
```

Comment:
I find this cleaner and easier to not forget down the line. Description from docs:

> UV_SYSTEM_PYTHON: Equivalent to the --system command-line argument. If set to true, uv will use the first Python interpreter found in the system PATH. 
> WARNING: UV_SYSTEM_PYTHON=true is intended for use in continuous integration (CI) or containerized environments and should be used with caution, as modifying the system Python can lead to unexpected behavior.

Reply:
If this is set `UV_SYSTEM_PYTHON=true`, then:`MODIN_ENGINE=python` is not needed?

Reply:
Redirecting pytest to the tubular pyproject.toml solved all the issues for tubular (`pytest tests --config-file=pyproject.toml`) 

`UV_SYSTEM_PYTHON=true` allows us to not specify `--system` each time when using `uv` to install dependencies

Reply:
This was making marimo ci fail. I rolled back all the changes except for adding tubular in downstream test and readme.